### PR TITLE
Nakul/mem 261

### PR DESCRIPTION
**Pull request by nakulbansal1609**
Made changes as per #todo comments and deleted the unwanted files after merging their codes.

### CODE REVIEW

**Comment by aby2s** [comment]
It would be good to make all tests pass after fixes.

File: meeshkan/nlp/schema_similarity/fields_similarity.py [hunk 1]
```diff
@@ -36,6 +40,64 @@ def similarity(self, a: typing.Set[str], b: typing.Set[str]) -> float:
...
+    def generate_nlp_vector(self, tokens_list: typing.List):
+        tokens_list = self.keep_only_alpha(tokens_list)
+        # tokens_list = self.camel_case_split_list(tokens_list)
+        tokens_list = self.convert_lower_list(tokens_list)
+        tokens_list = self.find_lemma_list(tokens_list)
+        tokens_list = self.remove_duplicate_and_sort(tokens_list)
+
+        return self.sentence_vector(tokens_list)
+
+    def cosine_distance(self, a, b):
```
Comment (aby2s) [inline meeshkan/nlp/schema_similarity/fields_similarity.py:87]: This class calculates similarity.

File: tests/conftest.py [hunk 2]
```diff
@@ -19,7 +20,7 @@ def nlp():
 
 @pytest.fixture()
 def opbank_spec():
-    with open("tests/resources/op_spec.json") as f:
+    with open("tests/r" "esources/op_spec.json") as f:
```
Comment (aby2s) [inline tests/conftest.py:23]: Is it an error?

File: meeshkan/nlp/schema_similarity/fields_similarity.py [hunk 3]
```diff
@@ -36,6 +41,68 @@ def similarity(self, a: typing.Set[str], b: typing.Set[str]) -> float:
...
+    def join_into_sentence(self, tokens_list: typing.List, separator=" ") -> str:
+        return separator.join(tokens_list)
+
+    def sentence_vector(self, tokens_list: typing.List):
+        sentence = self.join_into_sentence(tokens_list)
+        return self._nlp(sentence).vector
+
+    def generate_nlp_vector(self, tokens_list: typing.List):
+        tokens_list = self.keep_only_alpha(tokens_list)
+        #tokens_list = self.camel_case_split_list(tokens_list)
```
[the remark this answers is not in the document]
Comment (nakulbansal1609) [reply meeshkan/nlp/schema_similarity/fields_similarity.py]: @aby2s . I thought to import this function later once Maria will commit her chanes.

File: tests/conftest.py (hunk 2 above)
Comment (nakulbansal1609) [reply tests/conftest.py:23]: @aby2s . I am not sure how it got chaged. Yeah! this is an error.

File: meeshkan/nlp/schema_similarity/fields_similarity.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (aby2s) [reply meeshkan/nlp/schema_similarity/fields_similarity.py]: Yep, let's do it after Maria finishs

**Pull request merged**